src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const standartCompare = (a, b) => {
+    const strA = `${a}`;
+    const strB = `${b}`;
+
+    if (strA > strB) {
+      return 1;
+    } else if (strA === strB) {
+      return 0;
+    } else {
+      return -1;
+    }
+  };
+
+  [].__proto__.sort2 = function(compareFunction = standartCompare) {
```

Comment:
I think that using external variables in a function is a bad practice. Also you can simplify this.
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => a.toString() > b.toString() {
```

Reply:
oh really, thanks!